### ISSUE TITLE
Pack `HybridWebView` associated JS script as an embedded resource quite like `BlazorWebView` JS script

### ISSUE THREAD

**Issue by egvijayanand** [dotnet/maui#24730]
### Description

Pack `HybridWebView` associated JS script as an embedded resource quite like `BlazorWebView` JS script.

And standardize the name of the script file.

### Public API Changes

Same approach followed for the BlazorWebView JS.

### Intended Use-Case

This simplifies the distribution of script updates by just referencing them in the HTML page and updating the NuGet packages.

**Comment by Eilon** [comment]
Yeah definitely something we want to do. The experimental version had it, but it's out of scope for right now.

**Comment by mattleibow** [comment]
And we should also write this as typescript and/or create a .d.ts file so it is nice to use.

This is what I got so far:

```ts
// HybridWebView.d.ts

interface HybridWebView {
    SendRawMessage(message: string): void;
    InvokeDotNet(methodName: string, paramValues?: any): Promise<any>;
}

declare global {
    interface Window {
        HybridWebView: HybridWebView;
    }
}

export {};
```

**Comment by Eilon** [comment]
Note that we should update the JS code to be more idiomatic. Here's some feedback we got on the code style, patterns, etc.:

https://github.com/dotnet/maui/commit/503dbbe38970016f287b3aa3ee504ce2da0fd380#diff-0d85e38e166d3a7fedf8064ef350c6decc34f59e4a3d353732966ebcf18fdf7eR170-R175

(Note that the code isn't complete, and has some comments that need to be addressed.)

**Comment by mattleibow** [comment]
This will be in the next-ish net9 SR and net10 Preview 5
